packages/terra-tabs/src/Tabs.jsx

```diff
@@ -62,6 +66,10 @@ const propTypes = {
    * Callback function triggered when tab is drag and dropped.
    */
   onTabOrderChange: PropTypes.func,
+  /**
+   * A callback function triggered when a tab is closed.
+   */
+  onTabClose: PropTypes.func,
```

Comment:
The props required to show the add button (`onSelectAddButton` and `ariaLabelAddTab`) should be declared here

Reply:
Added https://github.com/cerner/terra-framework/commit/eba857452cdb936a451cc229d605f0db4d53a97c

Reply:
@saket2403  why does on tabclose is required to show add tab option ?